### DISCUSSION TITLE
crypt加密的视频文件在线播放的时候不能跳进度

### DISCUSSION THREAD

**Discussion by Drizzle365** [AlistGo/alist#5629] [Q&A]
不点击进度条可以正常播放，跳进度播放会一直显示加载中，加载不出

加密视频文件存在阿里云盘上，如果点击还没加载部分的进度条后会一直显示加载中，这是不可以从中间加载播放嘛
不知道这个问题是不是可以被解决的🥲

**Comment by KnIfER** [comment]
是这个吗，不支持跳进度 ? mp4 和 flv都不能吗，换个外部的播放器也不能吗

https://github.com/traceless/alist-encrypt
